blackbelt/dependencies.py

```diff
@@ -69,14 +69,22 @@ def check(dep, list_path, licenses_path, dev=False, debug=False):
     click.echo('Analyzing the package...')
     dep_name, dep_version = dep
 
-    with tempfile.TemporaryDirectory() as tmp_dir:
-        try:
-            install(dep_name, dep_version, tmp_dir, dev=dev)
-        except Exception as e:
-            if debug:
-                raise
-            raise click.BadParameter('The npm package could not be installed')
-        licenses = license_checker(tmp_dir)
+    if dep_name == '.': # check the `pwd` project deps
+        project_dir = '.'
+        package_json = os.path.join(project_dir, 'package.json')
```

Comment:
The behavior here is slightly different from the `else` with temporary directory - in this case, Black Belt doesn't install anything and just takes whatever you have in `node_modules`. While logical and useful, I'd warn user this is happening:

```python
click.echo('The dependency is a local directory, using whatever is currently in node_modules')
```

Also, if there are no `node_modules`, we could either run `install` on top of the directory, or to explicitly fail with something like

```python
raise click.BadParameter('There are no node_modules to analyze in the project directory')
```

Reply:
Added the node_modules warning and check.